src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -322,12 +322,13 @@ public void orderConfirmation(ArrayList<KitRequest> kitRequests, long startDate,
                 if (confirmationList != null && !confirmationList.isEmpty()) {
                     for (ShippingConfirmation confirmation : confirmationList) {
                         try {
-                            processingSingleConfirmation(gbfResponse, kitRequests, confirmation);
+                            processingSingleConfirmation(gbfResponse, confirmation);
                         } catch (Exception e) {
                             logger.error("Could not process confirmation for " + confirmation.getOrderNumber(), e);
                         }
                     }
-                    DBUtil.updateBookmark(endDate, DBConstants.GBF_CONFIRMATION);
+                    DBUtil.updateBookmark(endDate, DBConstants.GBF_CONFIRMATION); //TODO can be removed because not used anymore
```

Comment:
I left it in for now in case we want to change it back to using the bookmark as a start point for the confirmation call